src/styles/blocks/card.scss

```diff
@@ -0,0 +1,120 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  transition: $transition-speed;
```

Comment:
The variable `$transition-speed` is used here, but it is not defined within this file. Ensure that this variable is defined in a global SCSS file or imported correctly to avoid compilation errors.